File: client/Client/Network/Messages/Game/TickUpdateMessage.cs
using System;
using System.Collections.Generic;

namespace LoadRunnerClient.Messages
{
    public class TickUpdateMessage
    {
        public const string TYPE = "TickUpdateMessage";

        private Dictionary<string, List<Object>> _changes;

        public TickUpdateMessage(Dictionary<string, List<object>> changes)
        {
            this._changes = changes;
        }

        public Dictionary<string, List<Object>> changes
        {
            get => _changes;
            set => _changes = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/game/TickUpdateMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.game;

import java.util.HashMap;
import java.util.List;

public class TickUpdateMessage {

	public static final String TYPE = "TickUpdateMessage";

	private HashMap<String, List<Object>> changes;

	public TickUpdateMessage() {
	}

	public TickUpdateMessage(HashMap<String, List<Object>> changes) {
		this.changes = changes;
	}

	public HashMap<String, List<Object>> getChanges() {
		return changes;
	}

	public void setChanges(HashMap<String, List<Object>> changes) {
		this.changes = changes;
	}
}
